openxr/Cargo.toml

```diff
@@ -28,6 +28,7 @@ libloading = { version = "0.8", optional = true }
 [dev-dependencies]
 ash = { version = "0.38", default-features = false, features = ["loaded"] }
 ctrlc = "3.1.5"
+ndk-glue = "0.7"
```

Comment:
Should this be `[target.'cfg(target_os = "android")'.dev-dependencies]`?

Reply:
good idea, fixed now